tests/foreman/cli/test_contentviews.py

```diff
@@ -142,6 +142,46 @@ def setUp(self):
                 {u'organization-id': TestContentView.org['id']},
                 cached=True)
 
+    def test_bz1184442(self):
+        """@Test: Delete content view and verify it was actually deleted from
+        hard drive.
+
+        @Feature: Content View
+
+        @Assert: Content view was deleted and pulp folder doesn't contain
+        content view files anymore
+
+        @BZ: 1184442
```

Comment:
like @elyezer mentioned, we add BZ number here only if it is blocked by the BZ.  May be you can add a note inside the test saying that this test is originally written to test BZ 1184442